engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -739,6 +871,35 @@ else if (propertyName.equals(entry.getKey())) {
   }
 
   @GET
+  @Path(PATH_LOGICAL_PLAN_MODULES + "/{moduleName}/properties")
```

Comment:
Did the changes to look for operator first and if no operator is found then check for the module with given name. and print the property of the selected component.